source/connection-monitoring-and-pooling/tests/pool-clear-close-in-use.yml

```diff
@@ -0,0 +1,42 @@
+version: 1
+style: unit
+description: Pool clear MUST support lazily closing in use connections
+poolOptions:
+  backgroundThreadIntervalMS: 100
+operations:
+  - name: ready
+  - name: checkOut
+  - name: checkOut
+  - name: clear
+    closeInUseConnections: true
+  - name: waitForEvent
+    event: ConnectionPoolCleared
+    count: 1
+    timeout: 1000
+  - name: waitForEvent
+    event: ConnectionClosed
+    count: 2
+    timeout: 1000
+  - name: close
+events:
```

Comment:
This test and others omit `ConnectionCheckedIn` events, which are typically part of the Connection lifecycle. We should include them here to ensure driver's don't accidentally deviate from that.